arviz/plots/posteriorplot.py

```diff
@@ -47,10 +49,6 @@ def posteriorplot(trace, varnames=None, figsize=None, textsize=None, alpha=0.05,
         Controls the number of bins, accepts the same keywords `matplotlib.hist()` does. Only works
         if `kind == hist`. If None (default) it will use `auto` for continuous variables and
         `range(xmin, xmax + 1)` for discrete variables.
-    skip_first : int
```

Comment:
Out of curiosity why was this functionality removed? Not needed?

Reply:
Nevermind figured it out. Coordinates replaces it

Reply:
Actually, that's being removed everywhere, with the reasoning that users can filter the first elements themselves instead.

Reply:
Is there a document somewhere showing the "Roadmap" for arviz or is it just in peoples head. I'm not very opinionated, more so am just curious where you got that information from

I agree with the approach though, xarray seems easy and users can remove the burn in period themselves. Just to note it here I think somewhere there should be documentation just explaining that the user should filter the burn in period, and showing how. This is largely to the benefit of people that are newer to MCMC techniques, the argument made it obvious that a user should consider this. Its removal makes the code base simpler, just more precarious for new folks.

Reply:
We just sketched out a document last week! I will try to convert to markdown and post soon. 

This particular decision came out while porting one of the early functions over, but is not documented anywhere. I think the API will stabilize once we finish converting existing functions to xarray, then getting rid of some old code, and making a release on PyPI.  I'd guess that release comes in the next ~month.